app/controllers/admin/import_ect/find_ects_controller.rb

```diff
@@ -1,6 +1,6 @@
 module Admin
   module ImportECT
-    class FindECTController < AdminController
+    class FindECTsController < AdminController
```

Comment:
We're only trying to find one ECT here so to me the singular makes more sense.

Reply:
That's fair enough.

My approach was "rip out all the stuff and see what sticks".

I'll leave the changes that necessitated any controller renaming 👍🏼